src/typography/src/Text.js

```diff
@@ -51,6 +62,7 @@ class Text extends PureComponent {
         fontFamily={theme.getFontFamily(fontFamily)}
         marginTop={finalMarginTop}
         {...textStyle}
+        className={cx(className, css ? gcss(css).toString() : undefined)}
```

Comment:
Because we were supporting pass-through props to `Box` we now need to grab `css` ourselves and convert it to a classname as this point.

Technically there could be other places that are impacted as well, but I hopefully covered them by converting `Text` (`Pane` is already converted to support this scenario).